src/components/Highlight/index.tsx

```diff
@@ -10,7 +10,11 @@ export interface HighlightProps extends StandardProps {
   /**
    * String that needs to be highlighted
    */
-  highlight: string;
+  highlight?: string;
+  /**
+   * Array that defines the indices to highlight, each element is an array of [start, end]
```

Comment:
Could you also mention `highlight` is ignored in the case of `matches` provided?